chains/evm/deployment/utils/operations/contract/function.go

```diff
@@ -11,4 +11,14 @@ type FunctionInput[ARGS any] struct {
 	ChainSelector uint64 `json:"chainSelector"`
 	// Args are the parameters passed to the contract call.
 	Args ARGS `json:"args"`
+	// TimelockAddress is the address of the timelock contract, if applicable for MCMS proposals
```

Comment:
Operations shouldn't be aware of this, MCM proposal construction should happen at the changeset level & only the changeset should be aware of these addresses.